### PR TITLE
Week Two Milestone Reached

### PR DESCRIPTION
**Pull request by ChaoticTeal**
The player can speak to two NPCs, one of whom offers a quest. The quest is completed by speaking to the other NPC and returning to the first.

### CODE REVIEW

**Comment by dantogno** [comment]
@PurpleNinjaLink Did you add any scripts here? Remember, if you're going to import a bunch of stuff, do that in it's own PR so I don't have to search through thousands of files to see what you actually authored.

**Comment by dantogno** [comment]
Also, good news, LFS is working! 🎊

**Comment by ChaoticTeal** [comment]
I didn't add any scripts here, no. There are a couple more changes I might like to make before the PR is finalized--I expected I would need to revise things before it was approved. Good that LFS is working, though! My only imports are in the specified commit.

**Comment by ChaoticTeal** [comment]
I have now added one script, which is largely comprised of repurposed code from the MovePlayer script. At some point, perhaps I should make a more general animation script which can be used by both the player and the NPCs.

**Review by dantogno — COMMENTED** [review]
How were you able to accomplish this just by changing the scene file?

**Comment by dantogno** [comment]
This is too many files, I can't review it. We can look at it when we meet. Be sure to not do this anymore 😜

**Comment by ChaoticTeal** [comment]
I'm not sure what you mean by too many files... when I selected only the most recent 3 commits, there were 38 changed files, most of which were animation and meta files. If I've done something wrong in that regard, I'm not sure what it was. And for stopping the player from moving while talking, there was a quick solution in the triggers and interactions tutorial to simply disable the movement script when a conversation started and enable it when the conversation ends. I thought that would be the cleanest option.

File: Assets/Scripts/AnimateNPC.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    private Animator animator;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        animator = GetComponent<Animator>();
+    }
+
+    /// <summary>
+    /// Sets the NPC's animation direction to face the player initiating the conversation
```
Comment (dantogno) [inline Assets/Scripts/AnimateNPC.cs:19]: Explain in the summary where / how this gets called

File: Assets/Scripts/AnimateNPC.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            animator.SetTrigger("Move Up");
+        else if (playerDirection == Vector2.left)
+            animator.SetTrigger("Move Left");
+        else if (playerDirection == Vector2.right)
+            animator.SetTrigger("Move Right");
+
+    }
+
+    /// <summary>
+    /// Creates a normalized directional vector from the distance between the NPC and player
```
Comment (dantogno) [inline Assets/Scripts/AnimateNPC.cs]: just explain this a bit more clearly if possible

Comment (dantogno) [reply Assets/Scripts/AnimateNPC.cs]: consider renaming. My understanding is that this returns a 1 unit vector 2 pointed from the npc to the player. So figure out how to say that / name it.

**Review by dantogno — CHANGES_REQUESTED** [review]

**Pull request merged**